style.css

```diff
@@ -351,3 +368,172 @@ textarea::placeholder {
 input::placeholder {
   color: #979493;
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    padding: 24px 150px;
+    height: 32px;
+  }
```

Comment:
- [ ] The navbar still has a very big height different from what we have in Figma design (72px). Please try to fix that.